webapp/components/_new/generic/ResponsiveImage/ResponsiveImage.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <img
+    v-if="image && image.url"
+    :srcset="srcSet"
+    :sizes="sizes"
+    :src="proxyApiUrl(image.url)"
+    :alt="image.alt"
+    @error="$event.target.style.display = 'none'"
+  />
+</template>
+<script>
+export default {
+  props: {
+    image: { type: Object, default: () => ({}) },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:59:32Z" title="Wednesday, April 1st 2020, 8:59:32 am +02:00">Apr 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,34 @@
+<template>
+  <img
+    v-if="image && image.url"
+    :srcset="srcSet"
+    :sizes="sizes"
+    :src="proxyApiUrl(image.url)"
+    :alt="image.alt"
+    @error="$event.target.style.display = 'none'"
+  />
+</template>
+<script>
+export default {
+  props: {
+    image: { type: Object, default: () => ({}) },
```


I guess this can't be a required prop because it fill error out while waiting for the backend to return the `image` associated with a `Post` or `User`, right?
Why not have the default null, like for `user` in `UserTeaser.vue`